cms/static/js/views/list_item.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * A generic view to represent an editable item in a list.  The item
+ * has a edit view and a details view.
+ *
+ * Children Must implement:
+ * - itemDisplayName (string): Display name for the list item.
+ * - createEditView (function): Render and append the edit view to the
+ *   DOM.
+ * - createDetailsView (function): Render and append the details view
+ *   to the DOM.
+ */
+define([
+    'js/views/baseview', 'jquery', "gettext", "js/views/utils/view_utils"
+], function(
+    BaseView, $, gettext, ViewUtils
+) {
+    'use strict';
+
+    var ListItem = BaseView.extend({
+        canDelete: false,
+
+        initialize: function() {
+            this.listenTo(this.model, 'change:editing', this.render);
+            this.listenTo(this.model, 'remove', this.remove);
+        },
+
+        className: function () {
+            var index = this.model.collection.indexOf(this.model);
+
+            return [
+                'group-configuration',
```

Comment:
Sorry, this was my fault because I pushed this down into list_item since I needed it for both content_group_item and group_configuration_item. Now that I better understand your structure, I realize this is wrong.

Can the subclasses supply the className(s) as a property? I'm not sure why there need to be so many here. The one I actually needed for the test was 'group-configurations-list-item-' + index.